### PR TITLE
[Fix] Update model metafile format

### PR DESCRIPTION
**Pull request by ly015**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Update metafile foramt.

## Modification

* re-define collection name, task name, and model name
* add paper title
* fix some config path errors in .md files

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml [hunk 1]
```diff
@@ -1,40 +1,42 @@
 Collections:
 - Metadata:
     Architecture:
-    - HRNet
-  Name: hrnet_animalpose
+    - HRNet (CVPR'2019)
```
Comment (zhouzaida) [inline configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml:4]: Do we need to add the `CVPR2019` here?

File: configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
+  Name: animal 2D Keypoint topdown_heatmap animalpose
   Paper:
-  - http://openaccess.thecvf.com/content_CVPR_2019/html/Sun_Deep_High-Resolution_Representation_Learning_for_Human_Pose_Estimation_CVPR_2019_paper.html
-  - http://openaccess.thecvf.com/content_ICCV_2019/html/Cao_Cross-Domain_Adaptation_for_Animal_Pose_Estimation_ICCV_2019_paper.html
+  - Title: Deep high-resolution representation learning for human pose estimation
```
Comment (zhouzaida) [inline configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml:7]: Why does the collection has two papers?

Comment (ly015) [reply configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml:7]: One collection is corresponding to a dataset and a method, each can involve multiple papers.

File: configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml (hunk 1 above)
Comment (ly015) [reply configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml]: Fixed

**Pull request merged**